web/static/web/css/style.css

```diff
@@ -120,6 +120,53 @@ li {
     background-color: var(--make-blue) !important;
 }
 
+.make-bg-blue.ui.label {
+    color: white;
+}
+
+.ui.make-bg-blue.ribbon.label.primary.enlarged {
```

Comment:
Is it necessary to define a new, custom `enlarged` class, instead of using one of the classes Fomantic-UI has already defined? (see https://fomantic-ui.com/elements/label.html#size) 🤔

Reply:
Probably not :-P I now removed the enlarged class, and instead use the Fomantic-UI classes :-) I wasn't aware of this at first.